omod/src/main/resources/webModuleApplicationContext.xml

```diff
@@ -22,4 +22,17 @@
 	<bean class="org.openmrs.ui.framework.StandardModuleUiConfiguration">
 		<property name="moduleId" value="${project.parent.artifactId}"/>
 	</bean>
+
+	<bean class="org.openmrs.ui.framework.page.GlobalResourceIncluder">
```

Comment:
What is the effect of not having this bean?

Reply:
@dkayiwa that Bean is including the resources of javascript that are needed globally. For example the js that is needed to show the completion of patientQueue